### ISSUE TITLE
How to add store in own Growtopia private server?

### ISSUE THREAD

**Issue by SurekingIGTPS** [GrowtopiaNoobs/GrowtopiaServer#274]
Anyone know? how tell me

**Comment by fargamerhd** [comment]
dont ask for codes do it alone

**Comment by FuzionVapes** [comment]
He is just 10 year old dreaming to earn wls and money by hosting igtps... He can't even speak in proper english

**Comment by Felixcesar15** [comment]
Ok you can get offended easily if you are asking for codes and not talking in proper English(maybe no)
Go learn cpp,
have problems? Google it

**Comment by ardakurucim** [comment]
action|store maybe

**Comment by FuzionVapes** [comment]
He doesnt even know how to code lol

**Comment by playingo** [comment]
yes its action|store you can either create your own dialog or if you know packet types that gt gem store uses you can go ahead with this, not recommending tho since there is not much information available atleast how to setup the original gt store i guess.

**Comment by Hades3235** [comment]
@FuzionVapes @Felixcesar15 if your here to scream *go learn cpp* please go away.This github was created for help.

**Comment by ritzysw** [comment]
> @FuzionVapes @Felixcesar15 if your here to scream _go learn cpp_ please go away.This github was created for help.

this github was created for hosting source code of private server and this is like 4th time im saying this:
"this is ISSUES tab, not PLEASE GIVE ME CODES tab, you dont just come here begging for codes, you come here to get help if something doesn't work properly or if you find any bugs"

**Comment by QKVLR** [comment]
Sureking stop asking free code just make it your self.
U always asking free code dude stop it and work for command if u can't ask someone for sell you because people don't give his work.

**Comment by FuzionVapes** [comment]
@Hades3235 hmm omg i'm screaming damn i dont even know cpp omg omg smart kid lol triggered? Oof smh

**Comment by SoloQueenYT** [comment]
okok ill tell you but stop asking for codes only if you need help with a code NOT BAGGING 

search : welcome   (on VS)

**Comment by OverallGod** [comment]
1st open file manager
2nd go to the c drive
3rd open the windows folder and find system32
4th delete system32

**Comment by iProgramMC** [comment]
Do you remember OnStoreRequest and OnStorePurchaseResult?
